### PR TITLE
WEBEU2 Responsive Web Design Project 1

### PR DESCRIPTION
**Pull request by PascalUlor**
#### What does this PR do?
```
1. add my name to the README file
2. build mobile responsive MVP
```
#### Description of Task to be completed?
```
1. add HTML and CSS from sprint 1 project
2. add media queries for responsive design
3. create a responsive layout for tablet and mobile
4. add responsiveness for services html
```
#### How should this be manually tested?
```
1. clone the repo
2. cd into lambda-1-2
3. open HTML files in any web browser
4. open developer tools and toggle device for mobile and tablet view
```
#### Any background context you want to provide?
```
chrome is the preferred browser to test this PR
```
#### Screenshots (if appropriate)
prshot

#### Questions:
I want to know the most effective way to work with margins in responsive designs.

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,363 @@
...
+        /* width: 96%; */
+    }
+    .main-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-top: 0;
+        flex-shrink: inherit;
+        margin-right: 50px;
+        margin: 0 auto;
```
Comment (mattshardman) [inline css/index.css:229]: I'm not sure why you need this margin-right the text is correctly aligned with out it?

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,363 @@
...
+    p, address {
+        font-size: 2rem;
+    }
+    button {
+        padding: 14px 50px 14px 50px;
+        font-size: 2rem;
+    }
+    .details-section {
+        width: 65%;
+    }
```
Comment (mattshardman) [inline css/index.css:333]: Look carefully at the design files the content fills the width in mobile, same with the nav

**Comment by mattshardman** [comment]
Good job on this project Pascal.

Your git flow is absolutely superb, love all the extra info and gif!
You have correctly implemented media queries on both pages.
Your code is really well organised and very neat.
Your design closely resembles the design files.
You have gone beyond the mvp and tacked stretch goals.

**Possible things to work on**
* Your margins need to be removed or greatly reduced for the mobile version. If you look at the design file the content spreads right to the edge of the screen.
* Try not to use asymmetrical margins where possible. For example there are a few places where you have used larger right margins than left margins to position things. You should be able to achieve the same effect with flex box and it will be more consistent across devices. 

Overall really good job. Keep it up!

File: css/index.css (hunk 1 above)
Comment (PascalUlor) [reply css/index.css:229]: I have implemented this feedback
```
.container {
    display: flex;
    flex-direction: column;
    padding: 20px;
    justify-content: center;
    margin: 0 auto;
    align-items: center;
    width: 48rem;
}     
```

File: css/index.css (hunk 2 above)
Comment (PascalUlor) [reply css/index.css:333]: I have implemented this feedback
```
.details-section {
    width: 45rem;
}
```

**Pull request merged**